### PR TITLE
Portfolio detail popup windows PR

### PR DESCRIPTION
**Pull request by misikirdelelew**
I have done the detailed popup windows project by adding
index.html
style.css  and 
all dependencies

### CODE REVIEW

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @MishDM, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has done with pair but you did alone kindly implement it with your partner

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -409,6 +391,226 @@ ul {
...
+  padding: 16px;
+  gap: 12px;
+  position: fixed;
+  width: fit-content;
+  top: 20px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 16px;
+}
```
Comment (sumairq) [inline style.css:569]: - [ ] The popup is too wide, Please adjust it's proportions to make it look similar to figma design.
- [ ] The background should be blurred , just like in the figma template. 
- [ ] Add the icons for "See Live" and "See Source" links. 




Figma Design             |  Your Design
:-------------------------:|:-------------------------:
image  |  image

- [ ] Under the main popup heading , the sub headings are missing:
It should have sub headings like this 👇 

image

Kindly fix these issues and your project is good to go 👍

**Review by sumairq — CHANGES_REQUESTED** [review]
## Hi @MishDM , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Interactive elements are working properly
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️

- [ ] According to the figma design the project cards have alternating positions of the text and images, the first card has the image to the left and text to the right while the second card has an image to the right and the text to the left. Your project has all text to the right and the images to the left, kindly fix that. Cheers 👍 


Figma Design             |  Your Design
:-------------------------:|:-------------------------:
image  |  image


- [ ] There shouldn't be any horizontal scroll present in the webpage. Kindly fix this issue. Cheers 👍 

Template 1 portifolio

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -527,10 +741,86 @@ ul {
...
+    height: 70%;
+  }
+
+  .closeButton-class {
+    padding: 12px;
+    background-color: #fff;
+    background-image: url(./images/closeIcon.svg);
+    background-repeat: no-repeat;
+    float: right;
+    align-self: flex-end;
```
Comment (Ugboaja-Uchechi) [inline style.css:823]: - [ ] [OPTIONAL] Please remove the border around the `x-icon` 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @MishDM, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**